core/jazz_ui/src/app/pages/service-detail/service-detail.component.ts

```diff
@@ -126,18 +128,42 @@ export class ServiceDetailComponent implements OnInit {
         returnObject["eventScheduleRate"] = service.metadata.eventScheduleRate;
         returnObject["eventScheduleEnable"] = service.metadata.eventScheduleEnable;
         if(service.metadata.event_source){
+          if(service.platform === 'azure'){
+            if(service.metadata.event_source ==="dynamodb"){
```

Comment:
This looks incorrect. Why would event_source become dynamodb/kinesis/s3 etc when platform is azure?  Are we setting these incorrectly when we persist?

Reply:
These are for display purposes only. These dictate what you see on this screen: 
![image](https://user-images.githubusercontent.com/45375312/54796812-ff6b8380-4c0e-11e9-85af-e5f46b9303bd.png). Right now the code is very tightly coupled to aws deploymennt and I would be introducing changes that would impact internal stack and would require several days to acomplish. I wanted to avoid potentially breaking jazz internal.